.github/workflows/ci-failure-manager.yaml

```
@@ -0,0 +1,136 @@
+name: CI Failure Issue Manager
+
+on:
+  workflow_run:
```

actionlint (1.7.12)

[error] no workflow is configured for "workflow_run" event

(events)